### ISSUE TITLE
Cannot update Server via "MyServer" tab - [Windows]

### ISSUE THREAD

**Issue by hiimmazl** [Ylianst/MeshCentral#4298]
**Describe the bug**

The server cannot be update via the "My Server" tab. It is showing the current version directly, but stable and newest versions are "unknown".

**To Reproduce**
1. Go to 'My Server' tab
2. Try to update

**Screenshots**
![image](https://user-images.githubusercontent.com/14125915/180146824-c32235cc-6554-4705-825b-102ae101ba20.png)

**Server Software (please complete the following information):**
 - OS: Windows Server 2019
 - Virtualization: Virtual Machine on Hyper-V

 - Version: [e.g. 1.0.54]

**Your config.json file**
```
  "settings": {
    "cert": "meshcentral.obo.de",
    "_WANonly": true,
    "_LANonly": true,
    "_sessionKey": "",
    "port": 443,
    "aliasPort": 443,
    "redirPort": 80,
    "_redirAliasPort": 80,
	"npmpath": "C:\\Program Files\\nodejs\\node.exe C:\\Program Files\\nodejs\\npm",
	"npmproxy": "http://proxy:8080",
	"mongodb": "mongodb://127.0.0.1:27017/meshcentral",
	"mongodbcol": "meshcentral"
  },
```

**Comment by si458** [comment]
I belive your npm is incorrect
It needs to be something like `"npmPath": "c:\\npm.exe"`
The path to the npm.exe not the node.exe

**Comment by hiimmazl** [comment]
Thanks @si458 for the comment. I try to update my config file and test it!

I just used the information that were provided by @Ylianst via reddit : https://www.reddit.com/r/MeshCentral/comments/kzlzpy/comment/gjx1h54/?utm_source=share&utm_medium=web2x&context=3

It is hard to find a fully installation and customization guide for MeshCentral on Windows. The user and installation guides are helping, but they are missing very much information unfortunately. Maybe I need to check what I can commit to the documentation as I installed MeshCentral yesterday fresh and they were a few hiccups that were not explained directly only via reddit/forums etc..

**Comment by hiimmazl** [comment]
As I comment out the `npmpath`  the Update Tab is working correct. Somehow it did not work before, but after waiting about 1minute the correct versions are shown. 

So the update tab is working now without the mentioned setting, but I would be curious and glad if someone could tell me what the correct `npmpath` value would be if I want to set it directly via the config file!

**Comment by si458** [comment]
> As I comment out the `npmpath` the Update Tab is working correct. Somehow it did not work before, but after waiting about 1minute the correct versions are shown.
> 
> So the update tab is working now without the mentioned setting, but I would be curious and glad if someone could tell me what the correct `npmpath` value would be if I want to set it directly via the config file!

it would be something like `C:\\Program Files\\nodejs\\npm` if you installed it directly with the msi installer

also we are happy for you to create a tutorisl and we can add it to the docs! 
or
evem edit the docs yourself and create a PR!

@silversword411 is doing a great job with docs

also https://github.com/Ylianst/MeshCentral/blob/master/sample-config.json is the minimalist sample config.json you need to get meshcentral up and running

**Comment by hiimmazl** [comment]
Thanks for the fast feedback, but still not working with `C:\\Program Files\\nodejs\\npm`. NodeJS was installed previously by the Windows Installer. The MeshCentral itself was installed by myself via npm as the Windows Installer did succeed on NodeJS, but failed on MeshCentral somehow. 

As I documented everything I did (install ldap module, web-ssh module etc.) I will try to update the docs maybe by myself and do a PR for the windows part. 

Thank you very much for the help right now.
